heron/api/src/java/com/twitter/heron/api/Config.java

```diff
@@ -652,4 +661,31 @@ public void setTopologyStatefulCheckpointIntervalSecs(int secs) {
   public void setTopologyStatefulStartClean(boolean clean) {
     setTopologyStatefulStartClean(this, clean);
   }
+
+  /**
+   * Registers a timer event that executes periodically
+   * @param conf the map with the existing topology configs
+   * @param name the name of the timer
+   * @param timerDuration the frequency in which to run the task
+   * @param task the task to run
+   */
+  @SuppressWarnings("unchecked")
+  public static void registerTopologyTimerEvents(Map<String, Object> conf,
+                                                 String name, Duration timerDuration,
```

Comment:
I am wondering if "interval" is more clear than "timerDuration". Open to other thoughts.

Reply:
I will change the name